m2cgen/assemblers/boosting.py

```diff
@@ -19,6 +20,7 @@ def __init__(self, model, trees, base_score=0, tree_limit=None):
 
         assert tree_limit is None or tree_limit > 0, "Unexpected tree limit"
         self._tree_limit = tree_limit
+        self._total_num_leaves = total_num_leaves
 
```

Comment:
Will this value properly represent number fo leaves in case when we use `_assemble_multi_class_output` method? I have a feeling that we need to calculate number of leaves for every tree we pass into `_assemble_single_output` separately rather than having it as a class-level attribute.

Reply:
Will fix